package.json

```diff
@@ -1,22 +1,39 @@
 {
   "name": "trade-tariff-frontend",
   "private": true,
+  "dependencies": {
+    "@stimulus/polyfills": "^2.0.0",
```

Comment:
So all of these are actually development dependencies/shouldn't end up in the docker image

Reply:
Okey poke